packages/dpdk/Config.in

```diff
@@ -0,0 +1,291 @@
+if PACKAGE_libdpdk
+
+if x86_64
+config LIBDPDK_DRIVER_AXGBE
+	bool "axgbe driver support"
+	default n
+	help
+	  AMD axgbe Ethernet driver.
+
+config LIBDPDK_DRIVER_BNX2X
+	bool "bnx2x driver support"
+	default n
+	help
+	  QLogic 578xx Ethernet driver.
+
+config LIBDPDK_DRIVER_BNXT
+	bool "bnxt driver support"
+	default n
+	help
+	  Broadcom BCM5741X/BCM575XX Ethernet driver.
+
+config LIBDPDK_DRIVER_CNXK
+	bool "cnxk driver support"
+	default n
+	help
+	  Marvell OCTEON CN9K/CN10K Ethernet driver.
+
+config LIBDPDK_DRIVER_CXGBE
+	bool "cxgbe driver support"
+	default n
+	help
+	  Chelsio Terminator Ethernet driver.
+
+config LIBDPDK_DRIVER_DPAA
```

Comment:
Drivers like DPAA and others are on ARM too. It will be more universal to select in platform independent.

Reply:
Some of them are a little tricky...  The `igb` and `ixgbe` can either be on-chip or not, depending on the processor.

Reply:
Now I use with NXP DAAP. I don't think there will be a bug if all drivers are excluded from the x86-only limitation. Maybe it will change later, but for now it will be the easiest solution.